packages/database-model/lib/model.js

```diff
@@ -118,7 +118,7 @@ const DatabaseModel = AmpersandModel.extend(
       // if possible if we haven't fetched collections yet, but use the real
       // number if real collections info is available
       collectionsLength: {
-        deps: ['collection_count', 'collectionsStatus'],
+        deps: ['collection_count', 'collectionsStatus', 'collections'],
```

Comment:
It needs collections.length which is probably a missing dep, but I'm not 100% sure if we can depend on a collection.